docs/source/data/subscriptions.mdx

```diff
@@ -11,6 +11,7 @@ import {ExpansionPanel} from 'gatsby-theme-apollo-docs';
 
 Because subscription updates are usually pushed by the server (instead of polled by the client), they usually use [the WebSocket protocol](https://developer.mozilla.org/en-US/docs/Web/API/WebSockets_API) instead of HTTP. To support this, Apollo Server lets you [set a subscription-specific endpoint](#setting-a-subscription-endpoint) that's separate from the default endpoint for queries and mutations.
 
+// FIXME: not the case anymore
```

Comment:
I think we already know we're going to fix this file so we don't need to add FIXME comment (which will prevent this PR from being merged)